Assets/_BForBoss/_Core/Scripts/HUD/EnergySystemViewBehaviour.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using Perigon.Utility;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace BForBoss
+{
+    public class EnergySystemViewBehaviour : MonoBehaviour
+    {
+        [SerializeField] private Image _fillImage; 
+        private IEnergyDataSubject _energyDataSubject;
```

Comment:
Looks like there is no reason to cache `_energyDataSubject`

Reply:
Originally I was going to unsub from it ondestroy